contracts/base/Settings.sol

```diff
@@ -351,6 +359,66 @@ contract Settings is Pausable, SettingsInterface {
         );
     }
 
+    /**
+        @notice Add a new Node Component with its version.
+        @param componentName name of the component to be added.
+        @param minVersion minimum component version supported.
+     */
+    function addNewComponent(bytes32 componentName, uint256 minVersion)
+        external
+        onlyPauser()
+        whenNotPaused()
+    {
+        require(minVersion > 0, "INVALID_COMPONENT_MINIMUM_VERSION");
+        require(componentName[0] != 0, "MISSING_COMPONENT_NAME");
```

Comment:
I think it can be checked in this way:

require(componentName != "", "MISSING_COMPONENT_NAME")

Also, I think the error message could be "COMPONENT_NAME_MUST_BE_PROVIDED" (or similar) (just to be consistent with the rest of the errors). Please, verify the error message length is < 32 chars.

Reply:
Perfect, I already applied this suggestion.

Reply:
Suggestions implemented